Exiled.API/Features/Player.cs

```diff
@@ -1667,11 +1790,11 @@ public void ResetInventory(List<ItemType> newItems)
         /// Resets the player's inventory to the provided list of items, clearing any items it already possess.
         /// </summary>
         /// <param name="newItems">The new items that have to be added to the inventory.</param>
-        public void ResetInventory(List<Item> newItems)
+        public void ResetInventory(IEnumerable<Item> newItems)
```

Comment:
This is a breaking change.

Reply:
The entire PR is a breaking change, therefore I think it should be accepted if we want to enhance it regarding the implementation and compatibility with regard to the Item and Attachment system.